hw_steering_src/dr_common.py

```diff
@@ -0,0 +1,59 @@
+#SPDX-License-Identifier: BSD-3-Clause
+#Copyright (c) 2021 NVIDIA CORPORATION. All rights reserved.
+
+def dump_obj_str(keys, data):
+    _str = ""
+    length = len(keys)
+    for i in range(length):
+        key = keys[i]
+        if key == "id":
+            continue
+
+        if key == "mlx5dr_debug_res_type":
+            _str = _str + class_name_arr[data[key]]
+            if "id" in keys:
+                _str = _str + " " + data["id"]
+            _str = _str + ": "
+            continue
+
+        _str = _str + key + " "
+        if data[key] == None:
+            _str = _str + "NONE"
+        else:
+            _str = _str + data[key]
+        if i < (length - 1):
+            _str = _str + ", "
+        else:
+            _str = _str + "\n"
+
+    return _str
+
+
+
```

Comment:
2 empty lines, I see we have 2 empty lines for everything in master, lets keep it